src/checkers/inference/model/serialization/ToStringSerializer.java

```diff
@@ -230,13 +259,13 @@ public String serialize(ExistentialVariableSlot slot) {
     }
 
     @Override
-    public String serialize(CombVariableSlot slot) {
+    public String serialize(TernaryVariableSlot slot) {
         final StringBuilder sb = new StringBuilder();
         sb.append(slot.getId());
 
         if (showVerboseVars) {
             sb.append(": merges ");
-            sb.append(Arrays.asList(slot.getFirst(), slot.getSecond()));
+            sb.append(Arrays.asList(slot.getLeftOperand(), slot.getRightOperand()));
```

Comment:
`TernaryConstraint` and `TernaryVariableSlot` both have `leftOperant` and `rightOperand`, which is very confusing.